libs/product/state/src/reducers/configurable-product-entities/configurable-product-entities.reducer.ts

```diff
@@ -50,9 +50,11 @@ export function daffConfigurableProductEntitiesReducer<T extends DaffProduct, V
       );
     case DaffProductActionTypes.ProductLoadSuccessAction:
     case DaffProductPageActionTypes.ProductPageLoadSuccessAction:
```

Comment:
Same here as the comment above, since more than one of the incoming products could be configurable.